src/lib.rs

```diff
@@ -1,25 +1,45 @@
-use std::io::BufRead;
+use rayon::iter::ParallelBridge;
+use rayon::prelude::*;
```

Comment:
I prefer not to have `*` imports (also please separate the `std` and `rayon` import blocks, `std` first).

Reply:
Done.